extra-requirements.txt

```
@@ -32,8 +32,8 @@ grpcio-reflection>=1.46.0,<1.48.1:  core
 grpcio-health-checking>=1.46.0,<1.48.1:  core
 pyyaml>=5.3.1:              core
 packaging>=20.0:            core
-docarray>=0.16.4:           core
-jina-hubble-sdk>=0.30.4:   core
+docarray>=0.16.4,<=0.22.0:  core
```

```suggestion
docarray>=0.16.4,<=0.30.0:  core
```

Just in case we need to release docarray 0.22.1 for a bug fix on docarray v1

---

then put less than. Actually, I put 0.22 because right now docarray v1 is 0.21.0 so I was quite safe on that side